src/mscorlib/src/System/Text/UTF7Encoding.cs

```diff
@@ -170,20 +168,20 @@ public override int GetByteCount(char[] chars, int index, int count)
             return EncodingForwarder.GetByteCount(this, chars, index, count);
         }
 
-        [System.Runtime.InteropServices.ComVisible(false)]
+        [ComVisible(false)]
         public override int GetByteCount(String s)
         {
             return EncodingForwarder.GetByteCount(this, s);
         }
 
         [CLSCompliant(false)]
-        [System.Runtime.InteropServices.ComVisible(false)]
+        [ComVisible(false)]
```

Comment:
These ComVisible annotations should rather be deleted (ideally in separate PR). Similar to what was done in https://github.com/dotnet/corefx/pull/782 for CoreFX.